File: CSharp/Class/GetInstanceVariable.cs
using System;

// クラス定義
public class TypeFubuki
{
	// インスタンス変数
    public string name ="吹雪";
}

public class Test
{
        public static void Main()
        {
                // インスタンス生成
                TypeFubuki fubuki = new TypeFubuki();
                
                // 表示
                System.Console.WriteLine(fubuki.name); // 吹雪

        }
}

File: Java/Class/GetInstanceVariable.java
import java.util.*;
import java.lang.*;
import java.io.*;

// メイン
class Main
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// インスタンス生成
        TypeFubuki fubuki = new TypeFubuki();
        
        // コンソール出力
        System.out.println(fubuki.name); // 吹雪
	}
}

// クラス宣言
class TypeFubuki
{
    // インスタンス変数
    public String name = "吹雪";
 
}
